index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Aimal Amiri</title>
+
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@700&display=swap" rel="stylesheet" />
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <nav class="container">
+      <a href="#" class="navbar-logo font-inter text-bold decoration-none"> Aimal Amiri </a>
+      <a href="#" class="navbar-menu-button">
+        <img src="./assets/svg/menu.svg" alt="Menu" />
+      </a>
+    </nav>
+    <header class="header container">
+      <section class="header-content">
+        <h1 class="header-title font-crete-round text-orange">Hey There. I'm Aimal</h1>
+        <h2 class="header-subtitle text-white font-inter">I’m a Software Developer</h2>
```

Comment:
# Required Changes
- [ ] You are almost there with your structure. It would be nice you use just the `<section>` tag that serves as the container and gives it a class of header which you will style as the header section. Kindly remember the `<header>` tag is different from giving a section a class of `.header`.
- [ ] Please make your header section be in a section tag and then give it a class `.header-content` as you are already doing and remove the header tag on line 20 of your code.